simln-lib/src/test_utils.rs

```diff
@@ -196,10 +196,13 @@ impl LightningTestNodeBuilder {
 /// Creates a new simulation with the given clients and activity definitions.
 /// Note: This sets a runtime for the simulation of 0, so run() will exit immediately.
 pub fn create_simulation(clients: HashMap<PublicKey, Arc<Mutex<dyn LightningNode>>>) -> Simulation {
+    let (shutdown_trigger, shutdown_listener) = triggered::trigger();
     Simulation::new(
         SimulationCfg::new(Some(0), 0, 0.0, None, None),
         clients,
         TaskTracker::new(),
+        shutdown_trigger,
+        shutdown_listener,
```

Comment:
This needs to be in the commit where we refactor activity parser.

Reply:
:thinking: Do you mean in this commit? https://github.com/bitcoin-dev-project/sim-ln/pull/248/commits/7f5c6b3d557659cefe6d0a100517f2aaf08610f9.
Because this refactor was included in the next commits.